code/client/src/pages/Create.jsx

```diff
@@ -106,6 +107,23 @@ const Create = () => {
       message.error('Cannot deploy wallet. Error: root is not set.')
       return
     }
+    // Check for one address
+    let verifiedLastResortAddress = lastResortAddress
```

Comment:
Can we makes this a utility in util.js with boolean return value? Also, it should show some feedback to the user (e.g. message.error) at appropriate time (e.g. when full length is reached) when the address is incorrect. It should also give some visual signal when the full length is yet to be reached.